frontend/scripts/generateDocs.ts

```diff
@@ -1,24 +1,41 @@
 import fs from 'fs'
 import path from 'path'
+import { execSync } from 'child_process'
+
+const PACAKGE_NAME = '@acaldas/document-model-libs'
+
+try {
+  execSync(`npm ls ${PACAKGE_NAME}`)
+} catch (e) {
+  throw new Error(`Failed to generate documentation from the dependency (${PACAKGE_NAME}), did you run \`npm install\`?`)
+}
+
+// eslint-disable-next-line import/first
 import { concatMdSync } from 'concat-md'
```

Comment:
> Why was the import moved down here?

Since the generation runs before `npm run dev` - it is required to cover for the case where the `npm install` was not executed. I.e. if one removes `node_modules/` and runs a command that triggers generation - it has to be clear what is wrong.

> You know if wouldn't influence the order of events and concatMdSync will be imported before execSync is executed anyway, right?

I assume there's an error in the question and it is actually `You know it...` instead of `if...`. 

I disagree on the basis of the following evidence. 

This is the error if all the imports grouped together (`concat-md` is in the same batch as the others).

```

> powerhouse-switchboard-frontend@0.0.1 generateDocs
> vite-node ./scripts/generateDocs.ts

Failed to load url concat-md (resolved id: concat-md) in /Users/kirilldogadin/work/switchboard-boilerplate/frontend/scripts/generateDocs.ts. Does the file exist?
file:///Users/kirilldogadin/node_modules/vite/dist/node/chunks/dep-ca21228b.js:39470
        const err = new Error(`Failed to load url ${url} (resolved id: ${id})${importer ? ` in ${importer}` : ''}. ${msg}`);
                    ^

Error: Failed to load url concat-md (resolved id: concat-md) in /Users/kirilldogadin/work/switchboard-boilerplate/frontend/scripts/generateDocs.ts. Does the file exist?
    at loadAndTransform (file:///Users/kirilldogadin/node_modules/vite/dist/node/chunks/dep-ca21228b.js:39470:21) {
  code: 'ERR_LOAD_URL'
}
```

and this is the error if i run the generation as it is now

```
> powerhouse-switchboard-frontend@0.0.1 generateDocs
> vite-node ./scripts/generateDocs.ts

/Users/kirilldogadin/work/switchboard-boilerplate/frontend/scripts/generateDocs.ts:11
  throw new Error(`Failed to generate documentation from the dependency (${PACAKGE_NAME}), did you run \`npm install\`?`);
        ^

Error: Failed to generate documentation from the dependency (@acaldas/document-model-libs), did you run `npm install`?
    at /Users/kirilldogadin/work/switchboard-boilerplate/frontend/scripts/generateDocs.ts:10:9
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at ViteNodeRunner.directRequest (file:///Users/kirilldogadin/node_modules/vite-node/dist/client.mjs:308:5)
    at ViteNodeRunner.cachedRequest (file:///Users/kirilldogadin/node_modules/vite-node/dist/client.mjs:152:14)
    at ViteNodeRunner.executeFile (file:///Users/kirilldogadin/node_modules/vite-node/dist/client.mjs:132:12)
    at CAC.run (file:///Users/kirilldogadin/node_modules/vite-node/dist/cli.mjs:70:5)
```

P.S.

You also forgot to add a prooflink to this statement :D